services/server/src/validation/utils.ts

```diff
@@ -146,4 +146,5 @@ export const pageSize = zod.coerce.number().positive().min(1).int();
 export const pagination = zod.object({
```

Comment:
This `pagination` which equivalent to `IFilter.Pagination`. You should create another schema for `SkippablePagination`.